charts/datadog-monitors/values.yaml

```diff
@@ -0,0 +1,83 @@
+# -- (`string`) Optional shared pagerduty service name for monitors, will turn to a tag for alerts - if not provided, the .Release.name will be used by default
+serviceName:
+# -- (`string`) Optional shared pagerduty team name for monitors, will turn to a tag for alerts - if not provided, the tag will not be added
+team:
+
+# -- (`map[string]interface{}`) List of monitors
+monitors:
+  # -- (`map[string]interface{}`) Required: monitor resource name, Required unique monitor resource name(needed to allow value overrides and used a datadog monitor resource name)
+  resourceName:
```

Comment:
I would not create a real entry here and instead just have a large comment block. (Although I see you have disabled the entry below so I dont have strong opinions on this.)

Reply:
That was my original intention, but couldn't figure out how to properly format it so that helm-docs recognize the fields....